go.mod

```diff
@@ -12,6 +12,7 @@ require (
 
 replace (
 	github.com/coreos/etcd v3.3.10+incompatible => github.com/coreos/etcd v3.3.25+incompatible
+	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.1+incompatible
```

Comment:
Unrelated, fixing a vulnerability discovered in a dependency so nancy will be happy.